odoo/addons/test_orm/tests/test_properties.py

```diff
@@ -785,6 +785,111 @@ def test_properties_field_read(self):
             msg='Definition must be present when reading child')
         self.assertEqual(many2one_property['value'], (self.partner.id, self.partner.display_name))
 
+    def test_properties_field_html(self):
+        """Test that the HTML values are sanitized."""
+        xss_payload = "<img src='x' onerror='alert(1)'/>"
+        expected = '<img src="x">'
+        self.message_2.attributes = [
+            {
+                "name": "html",
+                "type": "html",
+                "string": "HTML",
+                "default": xss_payload,
+                "value": xss_payload,
+                "definition_changed": True,
+            },
+        ]
+        self.env.flush_all()
```

Comment:
Are all these flush_all needed? Could you remove them?
(I think only the one before assertRaises is needed, but this could be a separate test)